tensorflow/compiler/mlir/hlo/include/mlir-hlo/Dialect/mhlo/IR/hlo_ops.td

```diff
@@ -2307,9 +2307,8 @@ def HLO_RngBitGeneratorOp : HLO_Op<"rng_bit_generator", [NoSideEffect]> {
     https://www.tensorflow.org/xla/operation_semantics#rngbitgenerator.
   }];
   let arguments = (ins
-    // TODO(jpienaar): This could be an enum instead.
-    I32Attr:$rng_algorithm,
-    HLO_IntOrFpTensor:$initial_state
+    HLO_IntOrFpTensor:$initial_state,
```

Comment:
Why the order change? Prefer to keep this consistent with the argument order in operation semantics linked above.

Reply:
I have the following reasons:
1. attibuts always declare follow by operand.
2. xla hlo builder declare algorithm argument after state.

Reply:
I refer to the statement [here](https://github.com/tensorflow/tensorflow/blob/19025b6feacae8ef6636e0166e21ae5c2f04be64/tensorflow/compiler/xla/service/hlo_instruction.h#L566).I'm going to change those back.

Reply:
Yeah its one of those unfortunate things: the one you pointed out is "server side HLO" and considered internal. But it is confusing indeed.